### ISSUE TITLE
dump of zoobank bib ref records to POA

### ISSUE THREAD

**Issue by myrmoteras** [plazi/biblife#7]
@deepreef might it be possible to get @mguidoti a dump of the bibref records at Zoobank to be used to:
* prioritize journals for the data production to make its content accessible for GBIF, BLR
* to find out journals/articles that have no DOI, or are not in a repository so we can target them to move into BLR

**Comment by myrmoteras** [comment]
no . this will happen in  a couple of week, once rich will be at home ans has some time. But we could remind @deepreef

**Comment by deepreef** [comment]
Sorry for my delayed reply on this!  I am home now, but focused 100% on setting up database systems to allow our Museum staff to continue working from home.  This will keep me busy through at least the end of next week.  But once that is sorted out, I will be stuck at home myself working 100% on biodiversity database things, with this as my #1 priority.

Also, part of the tools I'm building to allow staff to work from home is a tool to start cleaning up people names.  I will use our dirty bucket of people names cross-linked to ZooBank clean-bucket names, with a tool very similar to what @myrmoteras used to clean up HNS journals to ZooBank/BHL/etc.

More on this in a couple of weeks.

**Comment by myrmoteras** [comment]
if you get into the person names, let's see how we could coordinate, or make use of, what the Canonical Names project does in which we are involved with data futures?!

**Comment by deepreef** [comment]
Yes, exactly -- that's the point.  For this week, I am focused on getting Bishop Museum staff working from home, which means the Canonical person names tools I'm building are focused on the dirty bucket I have (derived from Museum databases, mostly collector names but also author names).  However, I'm doing it in such a way that the tools I'm building will be helpful for our broader coordinated Canonical Names effort.  I will provide more details next week.

Should Peter Cornwell also join this Github group?
